homeassistant/components/jewish_calendar/__init__.py

```diff
@@ -32,8 +34,13 @@
     DEFAULT_NAME,
     DOMAIN,
 )
+from .sensor import INFO_SENSORS, TIME_SENSORS
 
 PLATFORMS: list[Platform] = [Platform.BINARY_SENSOR, Platform.SENSOR]
+PLATFORM_DESCRIPTIONS = {
```

Comment:
Make this a local variable in the function instead of a global constant.

Reply:
Done